pkgs/development/python-modules/nodriver/default.nix

```diff
@@ -11,14 +11,14 @@
 
 buildPythonPackage {
   pname = "nodriver";
-  version = "0.36";
+  version = "0.37";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "ultrafunkamsterdam";
     repo = "nodriver";
-    rev = "e630abfc5dce2023966a61cec739348b18bd465d";
-    hash = "sha256-pUWvHcsEPbRob5DDXBFOzqonSWigNPnPUHIu9omzYII=";
+    rev = "1bb6003c7f0db4d3ec05fdf3fc8c8e0804260103";
```

Comment:
Please ask for tags in an upstream issue.

Reply:
I'm not able to open an issue because the upstream repo owner restricted issues to collaborators only. @ToasterUwU do you happen to be/know any collaborators on [nodriver repo](https://github.com/ultrafunkamsterdam/nodriver) that could open an issue for tags ??

I can switch this to fetchPyPi as well, since there are no tests anyways, I think there is no downside. Plus it will be cleaner since it's versioned, instead of using the commit hash.

Reply:
> I can switch this to fetchPyPi as well, since there are no tests anyways

Sounds like a good idea.

Reply:
done: https://github.com/NixOS/nixpkgs/pull/343425